frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1129,19 +1128,42 @@ export const funnelLogic = kea<funnelLogicType>({
                 funnelStep: stepNumber,
             })
         },
-        openCorrelationPersonsModal: ({ entity, converted, resultType }) => {
-            personsModalLogic.actions.loadPeople({
-                action: { id: entity.id, name: entity.name, properties: entity.properties, type: entity.type },
-                label: entity.id,
-                date_from: '',
-                date_to: '',
-                filters: {
-                    ...values.filters,
-                    funnel_correlation_person_converted: converted ? 'true' : 'false',
-                    funnel_correlation_person_entity: entity,
-                },
-            })
-            eventUsageLogic.actions.reportCorrelationInteraction(resultType, 'person modal', { ...entity, converted })
+        openCorrelationPersonsModal: ({ correlation, success }) => {
+            if (correlation.result_type === FunnelCorrelationResultsType.Properties) {
+                const { breakdown, breakdown_value } = parseBreakdownValue(correlation.event.event)
+
+                personsModalLogic.actions.loadPeopleFromUrl({
+                    url: success ? correlation.success_people_url : correlation.failure_people_url,
+                    // I'm not sure why this is -2, I'm just copying from what was here previously
+                    stepNumber: success ? values.stepsWithCount.length : -2,
```

Comment:
don't need this anymore, it's already in the url now? (was https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/funnels/funnelLogic.ts#L1129 funnelStep, which we've overridden with `funnel_custom_steps` .

Reply:
These additional vars I kept as I thought they are needed for setting the title of the modal. I'll remove if not @neilkakkar 


If they are still needed, I'd rather keep this PR small. We could follow up with an addition to the people endpoint to include the specific filtering detail that could then be used for display, or have this to hand in an easy way in the initial response?

I'll add a NOTE about why it's still needed, unless you think we can remove them @neilkakkar

Reply:
Ah right, this tells whether it's success or failure? Cool cool